roles/basic_setup_rhel/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+enable_rhel7: true
+enable_rhel8: true
+sync_rhel7_now: true
+sync_rhel8_now: true
+wait_for_syncs: true
```

Comment:
Based on our new rules, I think this will, unforunately, need to be:

```
basic_setup_rhel_enable_rhel7: true
basic_setup_rhel_enable_rhel8: true
basic_setup_rhel_sync_rhel7_now: true
basic_setup_rhel_sync_rhel8_now: true
basic_setup_rhel_wait_for_syncs: true
```

CC @evgeni

Reply:
it'd be `foreman_…` tho ;)